src/components/guess.js

```diff
@@ -6,9 +6,14 @@ class Guess extends Component {
     const guess = this.props.guess;
     if (guess.get('status') === "correct") {
       let person = this.props.person;
+      let firstName = person.getIn(['0','first_name']);
+      let lastName = person.getIn(['0','last_name']);
+      let personId = person.getIn(['0','id']);
```

Comment:
Hm, I'm getting `undefined` when I run this locally with Flask. I think this needs to be `person_id` instead:

```suggestion
      let personId = person.getIn(['0','person_id']);
```

Reply:
You're right; I was looking at the RC API fields, and not the local database table column names. I updated the references from `id` to `person_id` to fix this. 

Based on our conversation today, I also made the update to only use `person_id` inside the directory link. Maybe a future task could use the `slug` field given that we found a couple of edge cases like [JP](https://www.recurse.com/directory/3181-john-paul-jp-ashenfelter) where his directory link is different from what we would expect from his displayed name.